source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,75 @@
+section: Publishing
+type: learn
+owner_slack: "#govuk-developers"
+last_reviewed_on: 2020-02-03
+review_in: 6 months
+related_applications: [content-tagger]
+---
+
+GOV.UK aims to organise content with a single taxonomy, [the topic taxonomy][topic-taxonomy], however due to legacy reasons there is a disassociated branch of this taxonomy known as the world taxonomy. The world taxonomy is used to tag guidance to particular world locations, for example [Latvia: doctors][latvia-doctors] is content tagged to the [Living in Latvia][live-latvia] section of the world taxonomy.
+
+In both the topic taxonomy and world taxonomy topics are tagged to taxons, those used for the world taxonomy are known as world location pages. World taxons are represented on GOV.UK as pages such as https://www.gov.uk/world/latvia, these are known as world location pages (despite them using the taxon [schema][]. The full list of world location pages can be seen on https://www.gov.uk/world.
+
+There are a small number of generic taxons that should be displayed on each of the individual world pages, for example [passports and emergency travel documents][passports]. These are then mirrored in the world pages, for example [passports and emergency travel documents Vietnam][vietnamese-passports], which has information specific to that country.
+
+You can view the hierarchy of the world taxonomy in [content-tagger][taxon-list].
+
+[topic-taxonomy]: https://docs.publishing.service.gov.uk/manual/taxonomy.html
+[latvia-doctors]: https://www.gov.uk/government/publications/latvia-list-of-medical-facilities
+[live-latvia]: https://www.gov.uk/world/living-in-latvia
+[schema]: https://github.com/alphagov/govuk-content-schemas
+[passports-generic]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[taxon-list]: https://content-tagger.publishing.service.gov.uk/taxons/91b8ef20-74e7-4552-880c-50e6d73c2ff9
+[passports]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[vietnamese-passports]: https://www.gov.uk/world/passports-and-emergency-travel-documents-vietnam
+
+## Differences between topic taxonomy and world taxonomy
+
+Whilst world taxons are treated in the same way as the regular [taxonomy][taxonomy] on a data level, they are a separate entity and do not share the same root. They cannot be found by visiting the root of the GOV.UK taxonomy meaning that they are simultaneously part of and not part of the core taxonomy.
+
+The two have very different structures. The world taxon structure has a breadth of over 200, one for each country and a few generic. The majority of these taxons have only one level below them while the rest have none. The topic taxonomy is an entirely different shape with 21 taxons under the GOV.UK home page each of which with a variable depth up to 5 or 6 levels.
+
+GOV.UK has 3780 published taxons as of the time of writing (03/02/2020), of those 2488 are world taxons meaning that they represent a substantial majority of all taxons.
```

Comment:
I think we can make this a little less specific and drop the date. For example could say "GOV.UK has approximately 4,000 taxons of these approximately 2,000 are part of the world taxonomy ..."

Reply:
Done :)